scripts/add_files.sh

```diff
@@ -2,6 +2,7 @@
 while read line;
 do
     cp -Rf src/.github $line;
+    sed -i "s/msName/$line/g" "src/Dockerfile";
```

Comment:
Change msName to microserviceName or microservice-name.
Don't want any confusion, or even worse, people think we have something to do with microsoft